dev-libs/sway/metadata.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>tajjada@gmail.com</email>
+		<name>Jasen Borisov</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription lang="en">
+		"SirCmpwn's Wayland window manager" is an i3-compatible window manager for Wayland.
+
+		sway was created to fill the need of an i3-like window manager for Wayland. The upstream i3 developers have no intention of porting i3 to Wayland, and projects proposed by others ended up as vaporware. Many thanks to the i3 folks for providing such a great piece of software, so good that your users would rather write an entirely new window manager from scratch that behaved *exactly* like i3 rather than switch to something else.
+	</longdescription>
+	<use>
+		<flag name="ffmpeg">Enable screencast (video grab) support in swaygrab, using ffmpeg.</flag>
```

Comment:
I'd much rather prefer this description than quite broad: Enable ffmpeg/libav-based audio/video codec support.